pkg/http/client_test.go

```diff
@@ -0,0 +1,74 @@
+package http
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"github.com/dcos/dcos-cli/pkg/config"
+	"github.com/stretchr/testify/require"
+)
+
+func TestGet(t *testing.T) {
+	query := `{"cluster":"DC/OS"}`
+	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.Write([]byte(query))
+	}))
+	defer ts.Close()
+
+	conf, err := config.FromString(`
+[core]
+
+dcos_url = "` + ts.URL + `"
+`)
+	require.NoError(t, err)
```

Comment:
Please use `conf := config.New()` and `conf.SetURL(ts.URL)`. Here we should focus on testing the http client's Get method.